Testscripts/Linux/dpdkUtils.sh

```diff
@@ -350,6 +350,15 @@ function Install_Dpdk () {
 		LogMsg "dpdk build with default DST IP ADDR on ${1}"
 	fi
 
+	# meson version on Ubuntu 16.04 is 0.29.0, on Ubuntu 18.04 is 0.45.1.
+	# dpdk meson version needs at least 0.47.1.
+	if [[ ${DISTRO_NAME} == ubuntu ]]; then
```

Comment:
For future maintenance, We should check for current meson version and take this decision.
Ubuntu might upgrade meson version and then we don't need to do this anymore.

Reply:
let me merge this PR, use new issue #1137 to address this.